<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Zqd</class>
 <widget class="QMainWindow" name="Zqd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zqd</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="fileDlgBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>FileDialog</string>
    </property>
   </widget>
   <widget class="QPushButton" name="colorDlgBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>ColorDialog</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fontDlgBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>FontDialog</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filePathedt">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="colorFrame">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="fontLineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>517</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
